tests/sentry/tasks/test_weekly_reports.py

```diff
@@ -281,6 +283,51 @@ def test_organization_project_issue_substatus_summaries(self):
         assert project_ctx.regression_substatus_count == 0
         assert project_ctx.total_substatus_count == 2
 
+    def test_organization_project_issue_status(self):
+        self.login_as(user=self.user)
+        now = timezone.now()
```

Comment:
I'm cautious about using times like this directly in code for flakiness and random stuff that happens. Would using freeze_time make sense here?